drs/drs-assembly/subscribe-model.conf

```diff
@@ -973,10 +673,10 @@ Information about current dither pattern position. Contains:
 
 {
   subsystem =  NFIRAOS
-  component =  nfiraos-tbd
```

Comment:
This RTC mode event has a whole bunch of information in it. It's not quite the same as your FITS keywords, so I think you will want to rethink the keywords to use more of the NFIRAOS terminology.